server/src/services/discord.service.ts

```diff
@@ -224,19 +224,19 @@ class DiscordService {
         }
         break;
       case 'space_join_request.created':
-        title = `${payload.member} requested to join ${payload.space}`;
+        title = `${payload.member.name} requested to join ${payload.space.name}`;
```

Comment:
I believe we should have a function called getEntityName and there we should handle like this:
getEntityName = (entity: Member | Space): string => entity?.name || entity